fudge-ratio/src/main/webapp/about_style.css

```diff
@@ -0,0 +1,53 @@
+@import url('https://fonts.googleapis.com/css2?family=Do+Hyeon&family=Open+Sans:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Bad+Script&family=Open+Sans:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&family=Poiret+One&display=swap');
+:root {
+  --background-color: rgb(224, 239, 241);
+  --font-color: rgb(30, 30, 30);
+  --accent-color:rgba(45, 93, 122, 0.712);
+}
+
+body { 
+  margin:0; 
+  background: var(--background-color);
+}
+
+.banner {
+  overflow: hidden;
+  /* 161, 111, 32, 0.712*/
+  background-color: rgba(45, 93, 122, 0.712);
+  font-family: 'Poiret One', cursive;
+  position: fixed;
+  top: 0;
+  width: 100vw;
+  height: 15vh;
+}
+
+#logo{
```

Comment:
Just a small suggestion: what do you think about putting all the classes on top and the ids in the bottom (or the opposite) to make it easier to find?

Reply:
I can do that !